analytics/src/analytics/integrations/etldb/migrations/versions/0005_add_proj_col_to_issue_hist.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE IF EXISTS gh_issue_history ADD COLUMN IF NOT EXISTS project_id INTEGER DEFAULT 0;
+ALTER TABLE IF EXISTS gh_issue_history ADD COLUMN IF NOT EXISTS sprint_id INTEGER DEFAULT 0;
```

Comment:
Similarly I'd be hesitant to set a default value for sprints. It's fine to just have this be `NULL` if an issue isn't assigned to a sprint.

If we really wanted to preserve history, we could populate this new column with data from `gh_sprint_map` on `(issue_id, d_effective)` but I think it's fine to leave them as `NULL`.

Reply:
Thanks for fixing that in [this commit](https://github.com/HHS/simpler-grants-gov/pull/2992/commits/68f20a389abed5c27b8c8074d169e7682cbec5f9)